doc/04_docker.md

```diff
@@ -58,13 +66,26 @@ Stops the same services and removes volumes & orphan containers:
 Defines:
 
 - Kafka broker (port 9092, 29092)
-- Kafka Connect with:
-  - Custom plugins in `/etc/kafka-connect/plugins`
+- Kafka Connect custom image with plugins:
+  - Builds and runs custom image
+  - Custom plugins in `/usr/local/share/kafka-connect/plugins`
   - Monitoring enabled via JMX Exporter
 - Connect waits for Kafka to be healthy before starting
 
 Topics are auto-created (`KAFKA_AUTO_CREATE_TOPICS_ENABLE=true`)
 
+Kafka Connect image is built from the [`Dockerfile`](../kafka-connect-custom/Dockerfile).
+
+Exposes:
+- Port `8083`: Kafka Connect API
+- Port `9100`: Prometheus metrics
+
+To build from `/kafka-connect-custom` directory you can use:
+
+```bash
+docker build --no-cache -t kafnus-connect:1.0.0 .
```

Comment:
```suggestion
docker build --no-cache -t kafnus-connect:devel .
```

or even

```suggestion
docker build --no-cache -t kafnus-connect .
```

We have not started to release named versions in Kafnus, so by the moment I'd suggest not using a number. `devel`, `latest`, etc. or no label at all seem to be better names.

Similar to what we do in https://github.com/telefonicaid/fiware-orion/tree/master/docker#3-build-a-docker-image, note that in that documentation we are not specifying any label to build the container.